59.md

```diff
@@ -6,157 +6,193 @@ Gift Wrap
-to a receiver's pubkey but there is no `p` tag pointing to the receiver. There is no way to know who the rumor is for
-without the receiver's or the sender's private key. The only public information in this event is who is signing it.
+A `seal` is a `kind:13` event that wraps a `rumor` and is signed with the sender's regular private key. The `seal` is **always** encrypted to a receiver's pubkey but there is no `p` tag to expose who can decrypt it. There is no way to know who the rumor is for without the receiver's or the sender's private key. The only public information in this event is who is signing it.
 
 ```js
 {
   "id": "<id>",
   "pubkey": "<real author's pubkey>",
-  "content": "<encrypted rumor>",
+  "created_at": now() or randomTimeUpTo2DaysInThePast(),
   "kind": 13,
-  "created_at": 1686840217,
   "tags": [],
+  "content": nip44Encrypt(rumor, "<recipient pubkey>", "<author's private key>"),
   "sig": "<real author's pubkey signature>"
 }
 ```
 
-Tags MUST must always be empty in a `kind:13`. The inner event MUST always be unsigned.
+`.content` contains the cyphertext of the [NIP-44](44.md) encryption of the JSON-encoded rumor. 
+
+Tags MUST must always be empty. 
+
+The inner event MUST always be unsigned.
 
-## 3. Gift Wrap Event Kind
+To increase privacy, `.created_at` MAY not be accurate. 
 
-A `gift wrap` event is a `kind:1059` event that wraps any other event. `tags` SHOULD include any information
-needed to route the event to its intended recipient, including the recipient's `p` tag or [NIP-13](13.md) proof of work.
+## The Gift Wrap Event Kind
+
+A `gift wrap` event is a `kind:1059` event that wraps any other event. `tags` SHOULD include any information needed to route the inner event to its intended recipient, including the recipient's `p` tag or [NIP-13](13.md) proof of work.
 
 ```js
 {
   "id": "<id>",
   "pubkey": "<random, one-time-use pubkey>",
-  "content": "<encrypted kind 13>",
+  "created_at": now() or randomTimeUpTo2DaysInThePast(),
   "kind": 1059,
-  "created_at": 1686840217,
-  "tags": [["p", "<recipient pubkey>"]],
+  "tags": [["p", "<recipient pubkey>", "<relay>"]],
+  "content": nip44Encrypt(event, "<recipient pubkey>", "<random private key>"),
   "sig": "<random, one-time-use pubkey signature>"
 }
 ```
 
-# Encrypting Payloads
-
-Encryption is done following [NIP-44](44.md) on the JSON-encoded event. Place the encryption payload in the `.content`
-of the wrapper event (either a `seal` or a `gift wrap`).
+`.content` contains the cyphertext of the [NIP-44](44.md) encryption of the JSON-encoded event. 
 
-# Other Considerations
+To increase privacy, `.created_at` MAY not be accurate. 
 
-If a `rumor` is intended for more than one party, or if the author wants to retain an encrypted copy, a single
-`rumor` may be wrapped and addressed for each recipient individually.
+Relays SHOULD only serve `kind 1059` events intended for the marked recipient based on user AUTH. 
```

Comment:
Please no, this breaks nip 87 groups. Or at least requires the user to AUTH using the shared key when fetching group events.

Reply:
interesting. I thought this was already in the text, but we can roll it back.

Reply:
Changed to MAY

Reply:
Ah, my bad, you're right. I guess we can leave it. I may have to re-think some things.

Reply:
We should get rid of it if it is getting on your way. I tried to soften the requirements to MAY, but let me know if it is still not what you need.

Reply:
We should probably get rid of it. It doesn't belong in this NIP at the very least. I could see gift wraps being created with no `p` tag, preferring instead some other way of addressing recipients. It definitely limits the extent to which this NIP can be used in creative ways.

At the same time, this is probably what relays will do by default, so implementers should probably think about how they can work around the recipient auth limitations 🤷‍♂️

Reply:
I know there have been other proposals to solve this. Maybe `-` is a good solution, but I'm not sure the nuances right now.